app/script_variant.py

```diff
@@ -29,6 +29,15 @@ def read_dictionary(file_name):
 JAPANESE_LOCATION_DICTIONARY = read_dictionary('jp_location_dict.txt')
 
 
+def is_kanji(word):
+    """
+    Returns whether word is kanji or not.
+    [\u3400-\u9fff]: kanji
```

Comment:
Nit: Please move this comment (about [\u3400-\u9fff]) out of docstring ("""...""") and make it a normal comment "#...". Docstring should comment the spec (what is this function and how to use it), not its internal implementation.